sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -483,11 +483,13 @@ inline pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
        (ur_device_info_t)UR_EXT_DEVICE_INFO_MAX_MEM_BANDWIDTH},
       {PI_EXT_ONEAPI_DEVICE_INFO_BFLOAT16_MATH_FUNCTIONS,
        (ur_device_info_t)UR_DEVICE_INFO_BFLOAT16},
-      {PI_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES,
+      {PI_EXT_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES,
+       (ur_device_info_t)UR_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES},
+      {PI_EXT_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES,
        (ur_device_info_t)UR_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES},
-      {PI_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES,
+      {PI_EXT_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES,
        (ur_device_info_t)UR_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES},
-      {PI_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES,
+      {PI_EXT_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES,
        (ur_device_info_t)UR_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES},
   };

```

Comment:
Since these "info" return a bit-set, I think we are missing a conversion (using converBitSet) of the result here: https://github.com/intel/llvm/blob/6490a36b6986e28fa4e5a786b4491a2cf47d8d5f/sycl/plugins/unified_runtime/pi2ur.hpp#L240

Reply:
Perhaps we need to explicitly handle all "info" parameters in `ur2piInfoValue` and avoid the conversions when not needed (for numerical values). This way new "info" conversions will less likely be missed (people will have to explicitly opt-out from it).

Reply:
@smaslov-intel : you mean to handle both input and output mappings, and have ur2piInfoValue and pi2urInfoValue?